images/win/scripts/SoftwareReport/SoftwareReport.Common.psm1

```diff
@@ -324,6 +324,13 @@ function Get-PipxVersion {
     return "Pipx $pipxVersion"
 }
 
+function Get-GDKVersion {
```

Comment:
nitpick: we can get this info from a registry key. Up to you.
```
$installedApplications | ? DisplayName -match "Microsoft Game" | select DisplayName,DisplayVersion

DisplayName                                                  DisplayVersion
-----------                                                  --------------
Microsoft Game Development Kit - 210602 (June 2021 Update 2) 10.1.19041.7011
```

Reply:
Agree. This would be better for consistency, since this approach already been used.
Updated